pkg/overlay/config.go

```diff
@@ -33,6 +33,7 @@ type LookupConfig struct {
 // NodeSelectionConfig is a configuration struct to determine the minimum
 // values for nodes to select
 type NodeSelectionConfig struct {
+	Version           string  `help:"set to minimum node software version" default:"v0.1.0"`
```

Comment:
oh wait this is going to break a lot of things, right? shouldn't we make selection by version opt-in?

Reply:
Thats what i think is never being used. Its always being overwritten by the peer setup